src/content/docs/tutorial-create-alerts/create-an-alert.mdx

```diff
@@ -13,37 +17,148 @@ import alertsThresholds from 'images/alerts_screenshot-crop_set-thresholds.webp'
+    alt="A screenshot displaying an alert condition created with a simple NRQL query"
+    src={alertsNRQL}
+    />
 
-1. Go to any dashboard. Then, find a chart you want to create an alert from, click the **...** menu, and select **Create alert condition**.
+1. From **one.newrelic.com**, select [**Alerts & AI**](https://one.newrelic.com/alerts-ai), then [**Alert conditions (policies)**](https://one.newrelic.com/alerts-ai/alerts-classic/policies).
 
-2. Select the signal, or stream of data, you want to use to create the alert condition. To keep things simple, select the **Build a query** option for your first condition.
+2. Select **New alert condition**.
 
-3. Modify the NRQL query to match what you want you to monitor, if necessary. For now, we recommend leaving the query as is, but once you have a better handle on alerts and NRQL queries, you can change the query to monitor nearly any data threshold.
+3. Write the NRQL query to create the condition. For example, if you wanted to monitor slow transactions, you can enter:
+```sql
+SELECT average(duration) from Transaction facet name
+```
 
 4. Select **Next** once you've finished to save your progress and move on to the next step.
-    </Step>
-    <Step>
+
+<Callout variant="tip">
+NRQL stands for New Relic's Query Language—a standardized query method that powers all our charts, and which you can use to analyze and query your data. If this is your first time working with NRQL, you may want to check out our [NRQL docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn more about NRQL, including [how it works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and an [introductory tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
+</Callout>
+        </TabsPageItem>
+        <TabsPageItem id="2">
+<SideBySide>
+    <Side>
+One of the easiest ways to get started is to use our **Golden signal or metric** option. When selected, this capability gives you a guided creation path for making a condition. All you have to do is select the options you want and let us do the rest!
+
+The following example will use specific options, but you can change whatever you need to create the exact alert condition that fits your specific needs.
+    </Side>
+    <Side>
+<img
+    title="golden metrics"
+    alt="A screenshot displaying the golden metrics alert creation option"
+    src={alertsGoldenCreate}
+/>
+    </Side>
+</SideBySide>
+<SideBySide>
+    <Side>
+<img
+    title="golden metric entities"
+    alt="A screenshot displaying the entities selection screen for golden metrics alerts"
+    src={alertsGoldenEntities}
+/>
+    </Side>
+    <Side>
+1. From **one.newrelic.com**, select [**Alerts & AI**](https://one.newrelic.com/alerts-ai), then [**Alert conditions (policies)**](https://one.newrelic.com/alerts-ai/alerts-classic/policies).
```

Comment:
IMO i would move steps 1-3 to the first chunk of text left of the screenshot. then i would make step 4 in the subsequent text section a new step 1. LMK if you have questions/this isn't clear

Reply:
Alternatively, you can make steps 1-3 into more exposition style description. That will fill the empty text in this first text description and eliminate some awkwardness from the next side by side where the text is longer than the image itself

Reply:
Good call, I changed the formatting of the side by side for this section and it should be more aesthetically pleasing and take up less room. Thanks very much!